KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/ModuleRTMPPublishDebug.java

```diff
@@ -0,0 +1,181 @@
+package com.kaltura.media.server.wowza;
+
+import com.wowza.wms.amf.*;
+import com.wowza.wms.application.*;
+import com.wowza.wms.client.*;
+import com.wowza.wms.logging.WMSLogger;
+import com.wowza.wms.logging.WMSLoggerFactory;
+import com.wowza.wms.logging.WMSLoggerIDs;
+import com.wowza.wms.module.*;
+import com.wowza.wms.request.*;
+import com.wowza.wms.stream.*;
+
+public class ModuleRTMPPublishDebug extends ModuleBase
+{
+	WMSLogger logger = null;
+	String category = WMSLoggerIDs.CAT_application;
+	String event = WMSLoggerIDs.EVT_comment;
+	String context = "";
+	
+	public void onAppCreate(IApplicationInstance appInstance)
+	{
+		logger = WMSLoggerFactory.getLoggerObj(appInstance);
```

Comment:
Tan-Tan, this code was sent from Wowza - they asked us to add this module temporarily for debug purposes so we don't want to change it